justfile

```diff
@@ -1,4 +1,5 @@
 set dotenv-filename := "integration_tests/.env.test"
+set dotenv-override := true
```

Comment:
This is actually something @Lun4m found out in a different branch but it affected me here. If you try to run this locally you want to export your own variables (and as such probably actually use the real S3 so the files get created there), if you then run the tests you want the test envs to be used not the local (potentially prod) ones.